src/Rector/Contrib/Symfony/DependencyInjection/ContainerBuilderCompileEnvArgumentRector.php

```diff
@@ -28,18 +28,18 @@ public function __construct(MethodCallAnalyzer $methodCallAnalyzer, NodeFactory
 
     public function isCandidate(Node $node): bool
     {
-        $isMethodCall = $this->methodCallAnalyzer->isTypeAndMethods(
+        if ($this->methodCallAnalyzer->isTypeAndMethods(
             $node,
             'Symfony\Component\DependencyInjection\ContainerBuilder',
             ['compile']
-        );
-
-        if ($isMethodCall === false) {
+        ) === false) {
```

Comment:
Why this change?

There is too much going on, resolving, validation and comparing to `false`. It's much less readable then before

Reply:
We do this everywhere, for example: https://github.com/rectorphp/rector/blob/master/src/Rector/Contrib/Symfony/DependencyInjection/CompilerAddPassPriorityRector.php#L42

Reply:
I see, ok. I'll try to upgrade my cyclomatix complecity for reading :)